codec/encode_test.go

```diff
@@ -1,9 +1,10 @@
 package codec_test
 
 import (
-	"github.com/jhump/protoreflect/codec"
 	"testing"
 
+	"github.com/jhump/protoreflect/codec"
```

Comment:
I see the previous import organization was wrong. But this is wrong, too. The "local" imports (e.g. same repo) should be in the third section (so below at line 9), not in a separate section.


Note to self: add `goimports` to CI and the `Makefile`.

Reply:
Done.